exporters/jaeger/include/opentelemetry/exporters/jaeger/recordable.h

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <jaeger_types.h>
+#include <opentelemetry/sdk/trace/recordable.h>
+#include <opentelemetry/version.h>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace jaeger
+{
+
+using namespace jaegertracing;
+
+class Recordable final : public sdk::trace::Recordable
```

Comment:
Nit: Not something which needs immediate attention, but probably we should have recordable names after transport types - eg `TUDPRecordable`. This is an issue with Zipkin exporter too.

Reply:
Agree. I logged the same comment for ETW exporter. I think we could create a task for the cleanup for all the exporters? I'll try the renaming anyway.

Reply:
Created issue https://github.com/open-telemetry/opentelemetry-cpp/issues/735 for this.